client/src/modules/vouchers/templates/action.cell.html

```diff
@@ -14,5 +14,10 @@
         <i class="fa fa-file-pdf-o"></i> <span translate>TABLE.COLUMNS.RECEIPT</span>
       </a>
     </li>
+    <li>
+      <a data-method="viewInvoice" ui-sref="journal({ filters : [{ key: 'record_uuid', value : row.entity.uuid}]})" href>
```

Comment:
This method probably shouldn't be `viewInvoice`, since you are in the voucher registry.